digestparser/utils.py

```diff
@@ -69,3 +70,8 @@ def url_quote(string):
         # python 3
         return urllib.parse.quote(string)
     return urllib.quote(string)
+
+
+def subject_slug(subject):
```

Comment:
smells like something that should be pulled in `elifetools` later right? The stopwords configuration is a classic configuration that could be centralized there to avoid the pain of extending it and forgetting some places